package-lock.json

```diff
@@ -0,0 +1,8358 @@
+{
```

Comment:
Please can we remove this file, this shouldn't be needed as we use `yarn.lock` instead.

Reply:
Oh yes, not sure how that got there - have removed it. 👍